content/en/integrations/guide/aws-integration-troubleshooting.md

```diff
@@ -59,10 +59,14 @@ When installing the Agent on an AWS host, you might see duplicated hosts on the
 
 ### EC2 metadata with IMDS v2
 
-In your [Agent configuration][5], if the parameter `ec2_prefer_imdsv2`, is set to `true` (defaults to `false`), the Agent requests EC2 metadata using Instance Metadata Service Version 2 (IMDSv2), which offers additional security for accessing metadata. In some situations, additional configuration may be required in AWS, for example: using a containerized Agent on a plain EC2 instance. See the [Transition to using Instance Metadata Service Version 2][6] documentation for details.
+EC2's [IMDSv2][8] can sometimes lead to problems gathering EC2 instance hostnames, where the Agent uses the `os` hostname provider instead of `aws`, as seen in the output of `agent status`.
```

Comment:
Should we develop a bit more ? This feels like there is a bug in the agent we don't want to fix. I think we should explain the issue and highlight that the agent fallback on OS  hostname (ie: it's not us it's them).

Reply:
This paragraph is an introduction to the section, so the remainder of the section does develop this idea more.  But I can shift this sentence around a little to sound more like "if you mess up IMDS, you can make it impossible for the agent to do its job".